routers/api/v1/org/team.go

```diff
@@ -221,6 +244,35 @@ func GetTeamMembers(ctx *context.APIContext) {
 	ctx.JSON(200, members)
 }
 
+// GetTeamMember api for get a particular member of team
+func GetTeamMember(ctx *context.APIContext) {
+	// swagger:operation GET /teams/{id}/members/{username} organization orgListTeamMember
+	// ---
+	// summary: List a particular member of team
+	// produces:
+	// - application/json
+	// parameters:
+	// - name: id
+	//   in: path
+	//   description: id of the team
+	//   type: integer
+	//   format: int64
+	//   required: true
+	// - name: username
+	//   in: path
+	//   description: username of the member to list
+	//   type: string
+	//   required: true
+	// responses:
+	//   "200":
+	//     "$ref": "#/responses/User"
+	u := user.GetUserByParams(ctx)
+	if ctx.Written() {
+		return
+	}
+	ctx.JSON(200, u.APIFormat())
```

Comment:
`u` maybe `nil`

Reply:
Wouldn't in that case `ctx.Written()` condition will evaluate to true and function will exit early?